ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -167,30 +170,12 @@ public void handleGet(RestRequest restRequest, RestResponseChannel restResponseC
 
   @Override
   public void handlePost(RestRequest restRequest, RestResponseChannel restResponseChannel) {
-    long processingStartTime = System.currentTimeMillis();
-    long preProcessingTime = 0;
     handlePrechecks(restRequest, restResponseChannel);
     restRequest.getMetricsTracker().injectMetrics(adminMetrics.postBlobMetrics);
```

Comment:
Same question as above. Moreover, I prefer to not have the post requests to take the metrics, as they will not go through the full cycle, and will create confusion when reading the metrics.